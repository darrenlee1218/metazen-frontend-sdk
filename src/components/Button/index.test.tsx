import React from 'react';
import TestRenderer from 'react-test-renderer';

import Btn from '.';

describe('Button', () => {
  test('renders correctly', () => {
    const element = TestRenderer.create(<Btn variant="outlined">Im a button</Btn>);
    expect(element.toJSON()).toMatchSnapshot();
  });
});
