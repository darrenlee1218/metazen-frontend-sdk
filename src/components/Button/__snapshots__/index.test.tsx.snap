// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1h2fv09-MuiButtonBase-root-MuiButton-root"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  Im a button
</button>
`;
